conans/client/conf/__init__.py

```diff
@@ -22,12 +22,14 @@
         version: ["7.0", "7.1", "8.0", "8.1", "8.2", "8.3", "9.0", "9.1", "9.2", "9.3", "10.0", "10.1", "10.2", "10.3"]
     FreeBSD:
     SunOS:
-arch: [x86, x86_64, ppc64le, ppc64, armv6, armv7, armv7hf, armv8, sparc, sparcv9, mips, mips64]
+    Arduino:
+        board: ANY
```

Comment:
In addition to this, as Arduino is not really an operating system there are some side effects, for example, if you build an algorithms library for the Arduino UNO board, it will work without any problem in any ATMega8U2 because you are not modeling any input output specific of the Arduino. 

Nevertheless, from the previous discussion we had about micro controllers and settings, probably we cannot aspire to model it perfectly because it's too complex and it hardly will be useful in central repositories but the company wide repositories.

So, for me, the question is, want we declare a settings structure that probably is too broad or too narrow for a concrete company knowing that hardly will adapt the open source needs? Could Arduino be an exception because of the good community behind to introduce the Arduino boards in the list? I have no feedback enough for Arduino/Conan so thanks for yours.